src/Pate/Verification/Simplify.hs

```diff
@@ -3,10 +3,13 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Is this extension actually used? I didn't see any type applications at a glance

Reply:
Removed